### ISSUE TITLE
Disc Priest

### ISSUE THREAD

**Issue by MartijnHols** [MartijnHols/WoWAnalyzer#58]
Use this issue to discuss features for this spec. Let us know when you're working on something, or plan to work on something.

**Comment by MartijnHols** [comment]
Currently being maintained by @MartijnHols.

**Comment by MartijnHols** [comment]
Bug: Nero implementation includes Atonement healing from all sources, instead of just from Penance.

**Comment by rp4rk** [comment]
We have a date for the new patch, Discipline has received a lot of changes, I want to have Discipline in a state to use on patch day:

## Mechanical Changes (Priority)
* [ ] Usage statistics for Power Word: Radiance charges.
* [ ] Evangelism
  * [ ] Typical atonement count on use
  * [ ] Healing done during Evangelism-seconds (?)
  * [ ] Failure to use Evangelism on cooldown (this will need to be reworded)

## Items
* [ ] Support for the new trinkets in the Tomb of Sargeras.
  * [ ] Archive of Faith (Generic)
  * [ ] Barbaric Mindslaver (Generic)
  * [ ] Archive of Faith (Generic)
  * [ ] Sea Star of the Depthmother (Generic)
  * [ ] The Deceiver's Grand Design (Generic)
  * [ ] Tarnished Sentinel Medallion
* [ ] March of the Legion
* [ ] New 2PC 
* [ ] New 4PC

**Comment by rp4rk** [comment]
#79 Implements a few of the trinkets on the list

**Comment by milesoldenburg** [comment]
We can close #21 because in 7.2.5 Plea will have a flat mana cost.

**Comment by rp4rk** [comment]
The parser is assuming castigation regardless of whether or not it is talented.

**Comment by rp4rk** [comment]
Atonements applied under Rapture with the t19 4pc will be considered as Evangelism healing.

Xalans interactions with Atonement expiration mean it is not compatible with how Evangelism is evaluated.

**Comment by rp4rk** [comment]
Evangelism CPM is monitored even without the talent selected.

**Comment by rp4rk** [comment]
* [ ] Atonements applied under Rapture with the t19 4pc will be considered as Evangelism healing.
* [ ] Xalans interactions with Atonement expiration mean it is not compatible with how Evangelism is evaluated.
* [ ] Evangelism CPM is monitored even without the talent selected.
* [ ] Include better analysis of Shadow Covenant effective healing.

**Comment by rp4rk** [comment]
* [ ] Remove absorbs from Twist of Fate module

**Comment by rp4rk** [comment]
* [ ] Borrowed Time implementation, BT counts as haste for the next cast, but is applied after player haste is considered. The GCD is reduced by BT as we.

**Comment by rp4rk** [comment]
[ ] T20 Two Set bonus is not working on live, it works on a local machine however. The problem seems to originate in the Penance module.

**Comment by MartijnHols** [comment]
I'd like to suggest adding an overview for the value of traits on relics. This may become even more useful in 7.3 with the NLC where you need to pick *two* traits instead of just one; example from the Holy Paladin version I did:

![image](https://user-images.githubusercontent.com/4565223/29812069-c05da602-8ca5-11e7-8954-4a0bd429c27b.png)

**Comment by rp4rk** [comment]
- [ ] Fix penance clipping regression
- [ ] Add traits

**Comment by Pastaaa** [comment]
Atm, if you are using Rupture during constant damage income (like wailing souls), you are able to see how much damage you absorb (cuz shields nearly instantly get broken).  
But if you are using Rapture for preshield reason and the real damage happens shortly after the Buff ends, the module in the cooldown tap can't show how much damage get absorbed, because it only tracks absorb during its buff duration. 

Example: I preshield the first hydra shots on Mistress Sasszine. I see how much absorb i applied and that it was very expensive (249K Mana) but i dont see, if it was worth, cuz the absorb happens after the Buff expires.

https://wowanalyzer.com/report/kBPFA7nQRpZ12fC8/33-Mythic+Mistress+Sassz'ine+-+Wipe+29+(5:47)/Past%C3%A0/cooldowns#spec-information


**TL:DR:** Showing how much of the applied absorb during rapture absorbs damage **during** and **after** the buff would be nice.
